source/driver.h

```diff
@@ -19,12 +19,12 @@ class Driver
      * This function read the parameter in "INPUT", "STRU" etc,
      * and split the MPI world into different groups.
      */
-    void reading();
+    static void reading();
```

Comment:
Why do you add static for this function?

Reply:
This is a check in clang-tidy. Please see https://clang.llvm.org/extra/clang-tidy/checks/readability/convert-member-functions-to-static.html